app/src/main/java/de/tum/in/tumcampusapp/component/tumui/lectures/model/LectureDetails.kt

```diff
@@ -0,0 +1,34 @@
+package de.tum.`in`.tumcampusapp.component.tumui.lectures.model
+
+import com.tickaroo.tikxml.annotation.PropertyElement
+import com.tickaroo.tikxml.annotation.Xml
+
+/**
+ * This class is dealing with the deserialization of the output of TUMOnline to
+ * the method "DetailsLehrveranstaltungen".
+ */
+@Xml(name = "row")
+data class LectureDetails(
+        @PropertyElement(name = "dauer_info") val duration: String? = null,
+        @PropertyElement(name = "ersttermin") val firstAppointment: String? = null, // TODO: TypeConverters
```

Comment:
Leftover TODO

Reply:
Upon further inspection, I saw that the value can also be “geplant” (“planned”). For that reason, I’ll remove the comment.